### DISCUSSION TITLE
Cannot read properties of undefined (reading 'gzip')

### DISCUSSION THREAD

**Discussion by bjl-person** [umijs/umi#11994] [General]
![QQ截图20231225002147](https://github.com/umijs/umi/assets/62779259/7c882dfb-e2c1-4af7-8aff-5e667fa65144)
当我写完代码的时候在控制台出现这个报错，项目中没有使用gzip相关代码不知道是哪里出现的问题

**Comment by 2362237588** [comment]
我也想问！我也是遇到了同样的问题

**Reply by youkei-zzz** [reply]
我也是

**Comment by fz6m** [comment]
升级浏览器到最新版本，然后关闭代理、开无痕再看看是不是浏览器插件的问题。

如有问题，需要给一个可以复现的仓库链接。
